classes/Link.php

```diff
@@ -804,6 +804,7 @@ public function getAdminLink($controller, $withToken = true, $sfRouteParams = ar
                     'AdminMeta' => 'admin_meta',
                     // 'AdminWebservice' => 'admin_webservice', @todo: uncomment when grid and entity form are done.
                     'AdminBackup' => 'admin_backup',
+                    'AdminContacts' => 'admin_contacts',
```

Comment:
not sure, but maybe it should be disabled until whole page is migrated? @matks wdyt?